Library/Homebrew/keg_relocate.rb

```diff
@@ -165,6 +165,45 @@ def replace_text_in_files(relocation, files: nil)
     changed_files
   end
 
+  def relocate_build_prefix(keg, old_prefix, new_prefix)
+    each_unique_file_matching(old_prefix) do |file|
+      # Skip files which are not binary, as they do not need null padding.
+      next unless keg.binary_file?(file)
+
+      # Skip sharballs, which appear to break if patched.
+      next if file.text_executable?
+
+      # Split binary by null characters into array and substitute new prefix for old prefix.
+      # Null padding is added if the new string is too short.
+      file.ensure_writable do
+        binary = File.binread file
+        odebug "Replacing build prefix in: #{file}"
+        binary_strings = binary.split(/#{NULL_BYTE}/o, -1)
```

Comment:
Do you use `NULL_BYTE` elsewhere? If not you may just want to make that constant a regex directly instead of into a string you interpolate into a `//o`.

Reply:
Unfortunately we do have to use `NULL_BYTE` as a string twice later in the method.  I suppose we could have a `NULL_BYTE_REGEX` to store the regex as a constant, but we only use it once so I'm not sure if that is necessary.